arangodb-net-standard/UserApi/UserApiClient.cs

```diff
@@ -53,7 +53,7 @@ public UserApiClient(IApiClientTransport client, IApiClientSerialization seriali
         /// <exception cref="ApiErrorException">ArangoDB responded with an error.</exception>
         /// <param name="body">The request body containing the user information.</param>
         /// <returns></returns>
-        public async Task<PostUserResponse> PostUserAsync(PostUserBody body)
+        public virtual async Task<PostUserResponse> PostUserAsync(PostUserBody body)
```

Comment:
What is the purpose of the virtual keyword in a class that is not derived?

Reply:
The virtual keyword is used to make the API methods extensible. Someone can then inherit from the class and override the specific methods they want to adapt to their need.

https://github.com/Actify-Inc/arangodb-net-standard/pull/254